index.md

```diff
@@ -29,7 +29,7 @@ This code is not exhaustive or complete. It serves to distill our common underst
 
 ### Diversity Statement
 
-We encourage everyone to participate and are committed to building a community for all. Although we may not be able to satisfy everyone, we all agree that everyone is equal. Whenever a participant has made a mistake, we expect them to take responsibility for it. If someone has been harmed or offended, it is our responsibility to listen carefully and respectfully, and do our best to right the wrong.
+We encourage everyone to participate and are committed to building a community for all. Although we may not be able to satisfy everyone, we all agree that everyone is worthy of respect. Whenever a participant has made a mistake, we expect them to take responsibility for it. If someone has been harmed or offended, it is our responsibility to listen carefully and respectfully, and do our best to right the wrong.
```

Comment:
No need to get defensive. If my wording offended you, I apologize.
We're talking subtleties here, which is only fair since your suggested change is a subtle one as well.

I have no problem with asking for respect; all I meant was to point out that replacing equality with respect seems to change the intended sentiment. 
Which led me to wonder what problem you had with equality that you thought "this should better be about respect instead".